code/processes/dqc.q

```diff
@@ -36,8 +35,8 @@ init:{
   `.dqe.configtable upsert .dqe.readdqeconfig[.dqe.configcsv;"S**SNNN"];
   update checkid:til count .dqe.configtable from `.dqe.configtable;
   /- from timespan to timestamp
-  update starttime:.z.d+starttime from `.dqe.configtable;
-  update endtime:?[0W=endtime;0Wp;.z.d+endtime] from `.dqe.configtable;
+  update starttime:($(.z.D,.z.d)gmttime)+starttime from `.dqe.configtable;
```

Comment:
If you are copying from the `getpartition` function I think you need to do

```
(`date^partitiontype)$(.z.D,.z.d)gmttime
```

Without the first part I don't think `$` throws an error.